catalogs/catalog-jdbc-common/src/main/java/org/apache/gravitino/catalog/jdbc/MySQLProtocolCompatibleCatalogOperations.java

```diff
@@ -47,6 +49,18 @@ public MySQLProtocolCompatibleCatalogOperations(
         columnDefaultValueConverter);
   }
 
+  @Override
+  public boolean checkJDBCDriverVersion() {
+    JDBCDriverInfo driverInfo = getDiverInfo();
+    if (driverInfo.majorVersion < MYSQL_JDBC_DRIVER_MINIMAL_SUPPORT_VERSION) {
```

Comment:
Do you mean we will not support MySQL driver 5.x?

Reply:
Yes, I found that our source code only uses the MySQL driver 8.x.  Do you think we still need to consider version 5.x. If we test one by one, it will be quite time-consuming.

Reply:
OK, but only 8.x is not enough. Some versions of 8.x have bugs. For more information, please see:
https://github.com/apache/gravitino/issues/3269

Reply:
I think we can restrict support for major versions like 5.x. For minor versions within 8.0, if there are bugs, we should fix them.

Reply:
Yeah.